src/api/optimize.c

```diff
@@ -38,7 +38,9 @@
 
 #include "cdirect.h"
 
+#ifdef WITH_LUKSAN
```

Comment:
use hte NLOPT_ prefix: WITH_LUKSAN => NLOPT_WITH_LUKSAN for consistency with NLOPT_CXX macro